README.md

```diff
@@ -301,6 +301,24 @@ To make it part of integration in addition to regression, add tag :integration;
 To exclude it from regression, add tag :non_regression (by default),
 or find the appropriate tag_to_exclude in config/mini_autobot/test_suite.yml
 
+#### Google Sheets
+
+As of v1.1, mini_autobot now supports automatically updating test plans stored in Google Sheets.
+
+To make this work, you will first need to set up OAuth2 access by setting up a project in the Google
+Developers Console and obtaining a client ID and client Secret.
+More information can be found here: https://developers.google.com/identity/protocols/OAuth2
+
+Place your client ID and client Secret in a new file at the following location:
+config/mini_autobot/google_drive_config.json
```

Comment:
A definite :+1: for having documentation, although another plus would be having a sample of the json file in the repository, e.g., naming it `config/mini_autobot/google_drive_config.sample.json`. The JSON should _look_ like the real one (i.e., the structure and key names), but with fake values—obviously, we don't want our real OAuth tokens to be out in the wild. :)

I don't think we have such a config directory for sample configs in this repository yet, so starting one wouldn't be a bad idea.